frontend/src/components/header.tsx

```diff
@@ -42,13 +43,18 @@ const SearchInputBase = styled(InputBase)({
   zIndex: 2,
 });
 
-
 export default function Header() {
+  const [expand, setExpand] = useState(true);
+
+  const handleDrawer = useCallback(() => {
+    setExpand(!expand);
+  }, [setExpand, expand]);

```

Comment:
```suggestion
  const [isExpanded, setIsExpanded] = useState(true);

  const handleDrawer = useCallback(() => {
    setIsExpanded(!isExpanded);
  }, [setIsExpanded, isExpanded]);

```

isExpaned & setIsExpanded로 표현하는게 더 명확할것 같다는 생각입니다. 검색해봐도 많이 쓰는 표현인것 같습니다 ㅎ
![5A77D952-064B-49E5-A04C-C02F6CB1356E](https://user-images.githubusercontent.com/36983960/153739669-3a1ec991-6567-4282-98f5-f70ef2ee7e1e.png)

Reply:
styled 에서 대문자를 사용할 수 없어서 expand 로 해야한다는데 동의합니다.